### DISCUSSION TITLE
Git code fixes bug but package not released yet, how do I use the latest code instead of package?

### DISCUSSION THREAD

**Discussion by fjur** [serverlessworkflow/specification#609] [Q&A]
I am dealing with a bug where I generate a serverless workflow DSL in javascript, then send it to a golang server and try and parse the DSL where it fails. It looks like the json generated as a boolean for the "exclusive" field which is causing the failure.
interface conversion: interface {} is bool, not *bool

The updated code seems to address this:
https://github.com/serverlessworkflow/sdk-go/blob/main/model/states.go#L161-L166

The latest release doesn't have this bug fix, whats the best way to circumvent this issue in the meantime so I can parse my workflow or get the latest version of the package? Will a new release occur soon?

**Comment by tsurdilo** [comment]
@ricardozanini hey, could we do a release for @fjur please? I know your going on pto, is there someone else that could do it in the meantime?

**Comment by ricardozanini** [comment]
@fjur I just did a release with the latest fixes provided by the community. I'm on vacation, away from my work computer. I don't have a Go setup. Please send a patch to fix this, and I will do a following up release?

**Comment by ricardozanini** [comment]
FWIW, here's the matrix for the SDK Go versions:
https://github.com/serverlessworkflow/sdk-go?tab=readme-ov-file#releases

We fixed the bugs and segregated them in GitHub tags so users can download the required version easily.
